clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -406,6 +448,19 @@ class ConstraintGenVisitor : public RecursiveASTVisitor<ConstraintGenVisitor> {
                                 TypeVarInfo &TVI)
       : Context(Context), Info(I), CB(Info, Context), TVInfo(TVI) {}
 
+  bool VisitTypedefDecl(TypedefDecl* TD) { 
+      CVarSet empty;
+      auto PSL = PersistentSourceLoc::mkPSL(TD, *Context);
+      bool shouldCheck = !PtrToStructDef::containsPtrToStructDef(TD);
```

Comment:
What does `shouldCheck` mean? Is there are a more evocative variable name you could choose?